test/cqlpy/test_tools.py

```diff
@@ -1401,7 +1423,6 @@ def test_json(self, query_template):
             sstable_query_result = json.loads(subprocess.check_output([
                 self._scylla_path, "sstable", "query",
                 "--logger-log-level", "scylla-sstable=debug",
-                "--reactor-backend=linux-aio",
```

Comment:
Did you deliberately remove this line? I thought @denesb only recently added it to avoid flaky test, I hesitate removing it now without a good reason.

Reply:
Indeed. I thought I fixed that, but apparently not.